.meta.toml

```diff
@@ -0,0 +1,20 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = "pure-python"
+commit-id = "b1a0ce9bae9fa73cb69c0d922eb73d55bd5c3080"
+
+[python]
+with-appveyor = false
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 55
+
+[manifest]
+additional-rules = [
+    "exclude appveyor.yml",
+    ]
```

Comment:
I wonder if zopefoundation/meta's `appveyor.yml` could be changed to use `zope.testrunner`?  It would be in line with its `tox` configuration, and as far as I can see this is the only substantive difference (there's some other setup stuff that AFAICS could be harmlessly brought into sync).  But I don't really know Appveyor at all.

Reply:
I suppose this comment actually belongs on `with-appveyor = false` above, but anyway.

Reply:
Wait, there's an `with-appveyor = false` somewhere?  Oops, I didn't realize.